application/common/models/User.php

```diff
@@ -521,22 +541,81 @@ public function getDisplayName()
         return $this->display_name ?? "$this->first_name $this->last_name";
     }
 
+    /**
+     * Based on provided time, determine whether to present a reminder to add
+     * an MFA option.
+     * @param int $now
+     * @return bool
+     * @throws Exception
+     */
+    public function isTimeToNagToAddMfa(int $now): bool
+    {
+        return MySqlDateTime::isBefore($this->nag_for_mfa_after, $now)
```

Comment:
I'm finding `isBefore` confusing here...I would think the code would read "is now after the nag date"...is this code accomplishing that same thing but in some dvorak keyboard way?

Reply:
Yes, but I'm not sure what could be different. Qwerty is much more confusing! Seriously, the only thing I can think of is to make a new `isAfter` method and swap the two arguments.

Reply:
a `MySqlDateTime.isAfter()` could be created and simply negate the other under the covers.  `MySqlDateTime::isAfter($now, $this->nag_for_mfa_after)` reads better to me but it's a nit if you don't think it's worth it.